### PR TITLE
Applied Sorting, Searching and pagination for Clients index page

### PR DESCRIPTION
**Pull request by Joeper214**

### CODE REVIEW

**Comment by Joeper214** [comment]
Please merge this PR after reviewing it. Thanks!

File: app/views/vue/_index.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@
       </tr>
     </thead>
     <tbody>
-      <tr class="infrastructure-row" v-repeat="
+      <tr class="{{option === 'infrastructure' ? 'infrastructure-row' : ''}}" v-repeat="
```
Comment (pocke) [inline app/views/vue/_index.html.erb]: Please use `v-class`.
http://vuejs.org/api/directives.html#v-class

**Comment by pocke** [comment]
LGTM except `v-class`.

File: app/views/vue/_index.html.erb (hunk 1 above)
Comment (Joeper214) [inline app/views/vue/_index.html.erb]: yes. thanks.

**Comment by pocke** [comment]
Merged. Thanks!

**Pull request merged**